README.md

```diff
@@ -1,33 +1,31 @@
-# Operate First template for repositories
+# Project Meteor
 
-Derive new repositories from this template
+Project Meteor is an AICoE team effort to build an end-end Data Science product showcasing the usefulness of Thoth and AI Ops capabilities along with the integration of  Open Data Hub/Red Hat OpenShift Data Science.
```

Comment:
please rephrase, dont think about what we want to achieve with it, think about what we do for... whom? ...

Reply:
updated the description